### ISSUE TITLE
typelevel/fs2: new versions not reported

### ISSUE THREAD

**Issue by satorg** [rtimush/sbt-updates#314]
Versions used:
  - SBT: v1.6.2
  - sbt-updates: v0.6.1

Consider this simple `build.sbt`:
```
scalaVersion := "2.13.8"

libraryDependencies ++= Seq("co.fs2" %% "fs2-core" % "3.2.6")
```
then run `sbt dependencyUpdates`:
```
~/projects/probes/dependency-updates-fs2$ sbt dependencyUpdates
[info] welcome to sbt 1.6.2 (Homebrew Java 11.0.15)
[info] loading settings for project global-plugins from plugins.sbt ...
[info] loading global plugins from /Users/satorg/.sbt/1.0/plugins
[info] loading project definition from /Users/satorg/projects/probes/dependency-updates-fs2/project
[info] loading settings for project dependency-updates-fs2 from build.sbt ...
[info] set current project to dependency-updates-fs2 (in build file:/Users/satorg/projects/probes/dependency-updates-fs2/)
[info] No dependency updates found for dependency-updates-fs2
[success] Total time: 0 s, completed May 24, 2022, 10:50:42 AM
```
So no updates found although there's definitely a newer version: [v3.2.7](https://github.com/typelevel/fs2/releases/tag/v3.2.7)

The plugin still works for all other libraries I got a chance to try.

**Comment by rtimush** [comment]
`sbt-updates` relies on the metadata files for maven repositories, and the [`maven-metadata.xml`](https://repo1.maven.org/maven2/co/fs2/fs2-core_2.13/maven-metadata.xml) for fs2-core doesn't actually contain `3.2.7`. This seem to be the same issue as in [MVNCENTRAL-6820](https://issues.sonatype.org/browse/MVNCENTRAL-6820) — the newer versions are most likely published to the "new" s01 Sonatype server, but it looks like there was a build of an older version that published artifacts to the "old" Sonatype server, and this screwed up the metadata. Hopefully, it will get fixed next time fs2 publishes any artifact.

That said, the same can potentially affect any library backporting fixes to older versions, so maybe I should make `sbt-update` check not only the metadata file, but also the [directory listing](https://repo1.maven.org/maven2/co/fs2/fs2-core_2.13/) for Maven Central as well.

**Comment by satorg** [comment]
Actually, I did more checks on that and noticed that the last FS2 version that sbt-updates still able to find updates for is "3.0.0" – it shows that "3.0.1" is available. But seems that all other subsequent versions are not detected by sbt-update anymore.

**Comment by rtimush** [comment]
Fixed in #317

**Comment by satorg** [comment]
Cool, thanks!
